src/services/groups.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Sequelize, Op } from 'sequelize';
+import { Groups as GroupsSchema, sequelize as sequelizeInstance } from '../data-access/postgresql';
+import Group from '../models/groups/group.types';
+import Permission from '../models/permission.type';
+
+const getAllGroups = async () => {
+    return await GroupsSchema.findAll();
```

Comment:
no need to use await in return statement, async function will return promise anyway

Reply:
done